dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -84,6 +85,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.tuple.Pair;
 import org.apache.http.HttpStatus;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I've avoided adding this in the past, even though the IDE has often tried to insert it. I'm not against it, just making sure it's intentional.

Reply:
Good point, I think the github library is using a javax annotation, so I'll remove it for now. (Not sure if it makes sense to override a javax annotation with a jakarta annotation)